app/src/test/java/ch/epfl/sdp/musiconnect/MusicianClassUnitTests.java

```diff
@@ -0,0 +1,277 @@
+package ch.epfl.sdp.musiconnect;
+
+import org.junit.jupiter.api.Test;
```

Comment:
This has to be ```import org.junit.Test``` (of course in every test file)

Reply:
@samuelchassot 
At the beginning I tried to use `import org.junit.Test` as you suggest but then I wasn't able to use the `assertThrows()` method to test all the exception throws. Then I figured out that with the jupiter package I was able to test them, hence I changed the import.
If this is not allowed, then what should I do to test the exception throws?

Reply:
Did you just try what I said here? Just replace all the ```import org.junit.jupiter.api.Test;``` by ```import org.junit.Test;```. That's it. No need to change your other imports.